ROADMAP.md

```diff
@@ -9,18 +9,18 @@ https://github.com/testground/testground/issues/1491
 
 ## Table Of Content
 
+- [Table Of Content](#table-of-content)
 - [Context](#context)
   - [About the Roadmap](#about-the-roadmap)
 - [Vision](#vision)
 - [Our Focus for 2022-2023](#our-focus-for-2022-2023)
 - [Milestones](#milestones)
   - [1. Bootstrap libp2p's interoperability testing story](#1-bootstrap-libp2ps-interoperability-testing-story)
-  - [2. Refresh Testground's EKS support](#2-refresh-testgrounds-eks-support)
-  - [3. Improve our Development & Testing infrastructure to meet engineering standards](#3-improve-our-development--testing-infrastructure-to-meet-engineering-standards)
-  - [4. Provide a Testground As A Service Cluster used by libp2p & ipfs teams](#4-provide-a-testground-as-a-service-cluster-used-by-libp2p--ipfs-teams)
-  - [5. Testground Is Usable by Non-Testground experts](#5-testground-is-usable-by-non-testground-experts)
-  - [6. Support libp2p's interoperability testing story and ProbeLabs work as a way to drive "critical" Testground improvements](#6-support-libp2ps-interoperability-testing-story-and-probelabs-work-as-a-way-to-drive-critical-testground-improvements)
-- [Appendix: Problems we focus on](#appendix-problems-we-focus-on)
+  - [2. Improve our Development \& Testing infrastructure to meet engineering standards](#2-improve-our-development--testing-infrastructure-to-meet-engineering-standards)
+  - [3. Support libp2p's interoperability testing story as a way to drive "critical" Testground improvements](#3-support-libp2ps-interoperability-testing-story-as-a-way-to-drive-critical-testground-improvements)
+  - [4. Testground Is Usable by Non-Testground experts](#4-testground-is-usable-by-non-testground-experts)
+  - [5. Refresh Testground's EKS support](#5-refresh-testgrounds-eks-support)
+  - [6. Provide a Testground As A Service Cluster used by libp2p \& ipfs teams](#6-provide-a-testground-as-a-service-cluster-used-by-libp2p--ipfs-teams)
```

Comment:
Should we drop these 2 altogether? My thinking is that a lot can change until 2024 and it's hard to tell if EKS is going to be our top priority.

I'm fine with leaving them in too but the roadmap might be easier to grasp as a whole without so far-reaching milestones.

Reply:
To me the goal here is to re-prioritize to accommodate changes to hiring, remember the goal is:

Reply:
Cool, that's good enough reason to leave it in for me :) Thanks 🙇

I'd merge as-is then. And we can keep iterating as we clarify things with other teams.